nix/nixos-image.nix

```diff
@@ -33,7 +33,13 @@ in import (pkgs.path + "/nixos/lib/make-disk-image.nix") {
         and `quit` to stop the VM.
       '';
       documentation.doc.enable = false;
-      environment.systemPackages = [ pkgs.kmod pkgs.qemu pkgs.gcc pkgs.linuxPackages.bcc ];
+      environment.systemPackages = [ 
+        pkgs.kmod
+        pkgs.qemu
+        pkgs.gcc
+        pkgs.linuxPackages.bcc
+        pkgs.busybox
```

Comment:
I think busybox also comes with `modprobe` so we no longer need kmod above.

Reply:
8211459